.github/workflows/ci.yml

```diff
@@ -65,7 +65,8 @@ jobs:
           restore-keys: ${{ runner.OS }}-node-
       - run: npm ci
       - run: npm run build
-      - run: npm run unit-test
+      - run: npm run test:unit
+      - run: npm run test:types -- --target 4.5,5.0,current
```

Comment:
@jdeniau What you think? Locally `test:types` would test only on the installed TypeScript. The lowest supported version (`4.5`, resolves to `4.5.5`) is checked on CI. I would definitely recommend adding `5.0` (resolves to `5.0.2`), because each major brings in breaking changes. And currently installed.

`latest` could be added. It points to `typescript@latest` in NPM registry, hence it is a changing target. It might start failing with any PR unrelated, i.e. even changes in types. That’s somewhat unpredictable.

Reply:
`latest` or even `next` could work as a daily cron job. Just some food for though.